.github/workflows/release-please.yml

```diff
@@ -37,5 +37,15 @@ jobs:
       - name: Publish Containers
         run: |
           GIT_VERSION=$(git describe --tags --always --match 'v*')
+          GIT_VERSION_MAJOR=$(echo $GIT_VERSION | cut -d. -f1)
```

Comment:
Cool that's confirms my second option in this comment https://github.com/linz/topo-imagery/pull/299#discussion_r1071568374